client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -152,16 +160,41 @@ export default function WPCOMBusinessAT( { product }: Props ): ReactElement {
 
 	// Handles dispatching automated transfer.
 	const dispatch = useDispatch();
-	const initiateAT = React.useCallback( () => {
+	const initiateAT = useCallback( () => {
 		setShowDialog( false );
 		dispatch( initiateThemeTransfer( siteId, null, '' ) );
 	}, [ dispatch, siteId ] );
+
 	const eventName =
 		product === 'backup'
 			? 'calypso_jetpack_backup_business_at'
 			: 'calypso_jetpack_scan_business_at';
 	const trackInitiateAT = useTrackCallback( initiateAT, eventName );
 
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+
+	useEffect( () => {
+		// Detect when the transfer is complete to refresh the site data
+		if ( automatedTransferStatus === COMPLETE ) {
+			// Need to refresh the site data.
+			dispatch( requestSite( siteId ) );
+		}
+	}, [ automatedTransferStatus ] );
+
+	useEffect( () => {
+		if ( automatedTransferStatus === COMPLETE && isJetpack ) {
+			dispatch(
+				successNotice( translate( 'Jetpack features are now activated.' ), {
```

Comment:
I think this string would better if we change it to `Jetpack features activated` or `Jetpack features are now active`. Invoking @ChaosExAnima since I'm not a native english speaker.

Reply:
I personally feel we should be specific: `Jetpack Backup is now active`. That said, don't be afraid to tag `Needs Copy Review` :)